### ISSUE TITLE
vpshere_guest: vm_nic options is not applied on reconfigure

### ISSUE THREAD

**Issue by gsatliawol** [ansible/ansible#19106]
_From @abergman on November 15, 2014 15:49_

Issue type: Bug report
Ansible version: 1.7.2

Component Name:
vsphere_guest module

Summary:
vm_nic option in vsphere_guest does not reconfigure the networks a VM is connected to
Steps To Reproduce:

Have an existing VM connected to "VM Network".
Run an ansible-playbook which reconfigures the VM to connect it to VM Network 2.

Expected Results:

The VM should be connected to VM Network 2.

Actual Results:

The VM is still connected to VM Network. Modifications to CPU/memory seem to work, but I am unable to use the vsphere_guest module to reconfigure a VM's networking.

Thoughts:

It looks like the entire code for adding a NIC to an existing VM is missing in vm_reconfigure, line 491.

_Copied from original issue: ansible/ansible-modules-core#335_

**Comment by gsatliawol** [comment]
_From @cchurch on November 15, 2014 20:51_

FYI, previous report of this issue before the great modules repository split of 2014: https://github.com/ansible/ansible/issues/7666

**Comment by gsatliawol** [comment]
_From @aioue on May 22, 2015 18:19_

Proposed documentation fix until full reconfigure functionality is added to the module.
https://github.com/ansible/ansible-modules-core/pull/1403

**Comment by gsatliawol** [comment]
_From @Xat59 on July 6, 2016 16:9_

Hi guys and @cchurch !

It seems this feature does not work yet , how can I help you or temporary use a workaround ?

Thank you guys !

**Comment by gsatliawol** [comment]
_From @ansibot on August 1, 2016 3:54_

@chrrrles, @rhoop, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by gsatliawol** [comment]
_From @ansibot on September 13, 2016 2:32_

@chrrrles, @rhoop, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by gsatliawol** [comment]
Question for @chrrrles, @rhoop, other maintainers: Is this issue fixed in the 2.x branch (i.e. are we only working on backporting to 1.x)?  If fixed in 2.x the issue is no longer relevant to me (I am one of the bug reporters) and I will want to unsub from the updates... if this is relevant in both 1.x and 2.x, I will continue to follow as I definitely want to know when this gets fixed _somewhere_...

**Comment by gsatliawol** [comment]
_From @ansibot on October 6, 2016 16:44_

@chrrrles, @rhoop, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by gsatliawol** [comment]
_From @ansibot on October 22, 2016 10:7_

@chrrrles, @rhoop, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by gsatliawol** [comment]
_From @ansibot on November 6, 2016 10:42_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by gsatliawol** [comment]
_From @ansibot on November 22, 2016 10:49_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by gsatliawol** [comment]
_From @ansibot on December 9, 2016 19:57_

This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
cc @akasurde @dagwieers @dav1x @jctanner @nerzhul
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @warthog9
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by warthog9** [comment]
This should probably be closed out as this is duplicated in the newer #38953 and this is a pretty old reporting

**Comment by ansibot** [comment]
cc @ckotte
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ghost** [comment]
As original bug writer I second @warthog9 's wanting to close as dupe. This is for an obselete version.
